### PR TITLE
EZP-31849 content edit screen improve design to display validation errors

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31849
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/fieldType/edit/_base-field.scss [hunk 1]
```diff
@@ -45,35 +46,34 @@
...
-    .ez-data-source__input {
-        &.is-invalid {
-            border-color: $ibexa-color-danger;
-            background: $ibexa-color-warning-pale;
+        color: $ibexa-color-primary;
+        align-items: center;
+        box-sizing: border-box;
+
+        &:before {
+            content: '!';
```
Comment (dew326) [inline src/bundle/Resources/public/scss/fieldType/edit/_base-field.scss:60]: I'm not sure if have `!` in new mockups?

File: src/bundle/Resources/views/themes/admin/content/form_fields.html.twig [hunk 2]
```diff
@@ -109,6 +109,7 @@
                     </svg>
                 </span>
             {% endif %}
+            <div class="ez-break-line"></div>
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/content/form_fields.html.twig:112]: I'm not sure if adding some divs just to make a break line is a proper way.

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/js/scripts/fieldType/ezauthor.js [hunk 3]
```diff
@@ -195,7 +195,7 @@
                     `${SELECTOR_FIELD_NAME} .ez-data-source__input`,
                     `${SELECTOR_FIELD_NAME} .ez-data-source__label`,
                 ],
-                errorNodeSelectors: [`${SELECTOR_FIELD_NAME} .ez-data-source__label-wrapper`],
+                errorNodeSelectors: [`${SELECTOR_FIELD_NAME}`],
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/fieldType/ezauthor.js:198]: just `[SELECTOR_FIELD_NAME]`

File: src/bundle/Resources/public/js/scripts/fieldType/ezauthor.js [hunk 4]
```diff
@@ -206,7 +206,7 @@
                     `${SELECTOR_FIELD_EMAIL} .ez-data-source__input`,
                     `${SELECTOR_FIELD_EMAIL} .ez-data-source__label`,
                 ],
-                errorNodeSelectors: [`${SELECTOR_FIELD_EMAIL} .ez-data-source__label-wrapper`],
+                errorNodeSelectors: [`${SELECTOR_FIELD_EMAIL}`],
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/fieldType/ezauthor.js:209]: as above

File: src/bundle/Resources/public/js/scripts/fieldType/ezuser.js [hunk 5]
```diff
@@ -116,28 +116,28 @@
                 eventName: 'blur',
                 callback: 'validateInput',
                 invalidStateSelectors: [`${SELECTOR_FIELD_USERNAME} ${SELECTOR_LABEL}`],
-                errorNodeSelectors: [`${SELECTOR_FIELD_USERNAME} ${SELECTOR_LABEL_WRAPPER}`],
+                errorNodeSelectors: [`${SELECTOR_FIELD_USERNAME}`],
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/fieldType/ezuser.js:119]: same as in first files, there are four occurenes in this file

File: src/bundle/Resources/public/scss/fieldType/edit/_base-field.scss [hunk 6]
```diff
@@ -104,3 +104,5 @@
         }
     }
 }
+
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/fieldType/edit/_base-field.scss:107]: two new lines

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

**Review by GrabowskiM — APPROVED** [review]

File: src/bundle/Resources/public/scss/_forms.scss [hunk 7]
```diff
@@ -38,6 +38,11 @@ form:not(.form-inline) {
     }
 }
 
+.ez-break-line {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_forms.scss:41]: Seems like it is not needed anymore?

**Review by dew326 — CHANGES_REQUESTED** [review]

**Review by dew326 — APPROVED** [review]

**Pull request merged**